sagemaker-python-sdk/pytorch_mnist/pytorch_mnist.ipynb

```diff
@@ -156,12 +152,12 @@
     "\n",
     "estimator = PyTorch(entry_point='mnist.py',\n",
     "                    role=role,\n",
-    "                    framework_version='1.4.0',\n",
+    "                    framework_version='1.5.0',\n",
     "                    train_instance_count=2,\n",
-    "                    train_instance_type='ml.c4.xlarge',\n",
+    "                    train_instance_type='ml.p2.8xlarge',\n",
```

Comment:
the default instance has to be 'ml.c4.xlarge' based on availability by default in different regions.